tests/integration/hxl/metadata.feature

```diff
@@ -0,0 +1,88 @@
+@hxl
```

Comment:
This should probably have `@hxlEnabled` to ensure the feature is always enabled.